build downloaded release fails

**Bug Report**

What happened:
Building a downloaded release from https://github.com/ceph/ceph-container/releases fails because the build script expects to live in a git repository.

What you expected to happen:
Building a downloaded release works.

How to reproduce it (minimal and precise):
```
curl -J -L -O https://github.com/ceph/ceph-container/archive/v3.0.5.tar.gz
tar xf ceph-container-3.0.5.tar.gz
cd ceph-container-3.0.5
make FLAVORS="luminous,centos,7 luminous,ubuntu,16.04 mimic,centos,7" build
```
```
fatal: not a git repository (or any of the parent directories): .git
fatal: not a git repository (or any of the parent directories): .git
fatal: not a git repository (or any of the parent directories): .git

  CEPH_VERSION      : luminous
  CEPH_POINT_RELEASE:
  DISTRO            : centos
  DISTRO_VERSION    : 7
  HOST_ARCH         : x86_64
  BASEOS_REGISTRY   :
  BASEOS_REPO       : centos
  BASEOS_TAG        : 7
  IMAGES_TO_BUILD   : daemon-base daemon
  STAGING_DIR       : staging/luminous--centos-7-x86_64
  RELEASE           :
  DAEMON_BASE_IMAGE : ceph/daemon-base:-luminous-centos-7-x86_64
  DAEMON_IMAGE      : ceph/daemon:-luminous-centos-7-x86_64

Computed:
  BASE_IMAGE        : centos:7

fatal: not a git repository (or any of the parent directories): .git
fatal: not a git repository (or any of the parent directories): .git
fatal: not a git repository (or any of the parent directories): .git
fatal: not a git repository (or any of the parent directories): .git
Traceback (most recent call last):
  File "maint-lib/stage.py", line 116, in <module>
    main(CORE_FILES_DIR, CEPH_RELEASES_DIR)
  File "maint-lib/stage.py", line 103, in main
    copy_files(src_files, src_path, staging_path, files_copied)
  File "/Users/raboo/repos/ceph-container-3.0.5/maint-lib/stagelib/filetools.py", line 69, in copy_files
    dirty_marker = '*' if git.file_is_dirty(file_path) else ' '
  File "/Users/raboo/repos/ceph-container-3.0.5/maint-lib/stagelib/git.py", line 39, in file_is_dirty
    if re.match(r'^\s*' + file_path + '$', file_status_msg):
  File "/usr/local/Cellar/python/3.6.5/Frameworks/Python.framework/Versions/3.6/lib/python3.6/re.py", line 172, in match
    return _compile(pattern, flags).match(string)
TypeError: expected string or bytes-like object
make: *** [stage.x86_64,luminous,centos,7] Error 1
```

**Build Environment**:
* OS: OS X
* Kernel: Darwin 16.7.0
* Docker version: 18.03.1-ce

--- a/README.md
+++ b/README.md
@@ -25,6 +25,47 @@ $ curl -s https://registry.hub.docker.com/v2/repositories/ceph/daemon/tags/ | jq
 Be careful, by default the Docker API returns the first page with its 10 elements.
 To improve your `curl` you can pass the `https://registry.hub.docker.com/v2/repositories/ceph/daemon/tags/?page=2`
 
+Stable images
+-------------
+Since everyone doesn't use Docker Hub API and Docker Hub WebUI doesn't paginate. It's hard to see all available stable images.
+
+Here is a list of available stable Ceph images
+
+```
+ceph/daemon:v3.0.5-stable-3.0-luminous-centos-7
+ceph/daemon:v3.0.5-stable-3.0-jewel-centos-7-x86_64
+ceph/daemon:v3.0.5-stable-3.0-kraken-centos-7-x86_64
+ceph/daemon:v3.0.5-stable-3.0-kraken-ubuntu-16.04-x86_64
+ceph/daemon:v3.0.5-stable-3.0-jewel-ubuntu-16.04-x86_64
+ceph/daemon:v3.0.5-stable-3.0-jewel-ubuntu-14.04-x86_64
+ceph/daemon:v3.0.5-stable-3.0-luminous-centos-7-x86_64
+ceph/daemon:v3.0.5-stable-3.0-luminous-ubuntu-16.04-x86_64
+ceph/daemon:v3.0.5-stable-3.0-luminous-centos-7-aarch64
+ceph/daemon:v3.0.3-stable-3.0-kraken-ubuntu-16.04-x86_64
+ceph/daemon:v3.0.3-stable-3.0-jewel-centos-7-x86_64
+ceph/daemon:v3.0.3-stable-3.0-jewel-ubuntu-16.04-x86_64
+ceph/daemon:v3.0.3-stable-3.0-jewel-ubuntu-14.04-x86_64
+ceph/daemon:v3.0.3-stable-3.0-luminous-centos-7-x86_64
+ceph/daemon:v3.0.3-stable-3.0-luminous-ubuntu-16.04-x86_64
+ceph/daemon:v3.0.3-stable-3.0-kraken-centos-7-x86_64
+ceph/daemon:v3.0.2-stable-3.0-jewel-ubuntu-14.04-x86_64
+ceph/daemon:v3.0.2-stable-3.0-kraken-centos-7-x86_64
+ceph/daemon:v3.0.2-stable-3.0-jewel-ubuntu-16.04-x86_64
+ceph/daemon:v3.0.2-stable-3.0-kraken-ubuntu-16.04-x86_64
+ceph/daemon:v3.0.2-stable-3.0-luminous-centos-7-x86_64
+ceph/daemon:v3.0.2-stable-3.0-luminous-ubuntu-16.04-x86_64
+ceph/daemon:v3.0.2-stable-3.0-jewel-centos-7-x86_64
+ceph/daemon:v3.0.1-stable-3.0-jewel-ubuntu-16.04-x86_64
+ceph/daemon:v3.0.1-stable-3.0-kraken-centos-7-x86_64
+ceph/daemon:v3.0.1-stable-3.0-jewel-ubuntu-14.04-x86_64
+ceph/daemon:v3.0.1-stable-3.0-kraken-ubuntu-16.04-x86_64
+ceph/daemon:v3.0.1-stable-3.0-luminous-centos-7-x86_64
+ceph/daemon:v3.0.1-stable-3.0-luminous-ubuntu-16.04-x86_64
+ceph/daemon:v3.0.1-stable-3.0-jewel-centos-7-x86_64
+ceph/daemon:tag-stable-3.0-luminous-ubuntu-16.04
+ceph/daemon:tag-stable-3.0-luminous-centos-7
+ceph/daemon:tag-stable-3.0-jewel-centos-7
+```
 
 Core Components
 ---------------

